roles/multiqc/tasks/main.yml

```diff
@@ -1,7 +1,20 @@
 ---
+- name: Fetch MultiQC from GitHub
+  git: repo="{{ multiqc_repo }}"
+       dest="{{ multiqc_dest }}"
+       version="{{ multiqc_version }}"
+       force=yes
+  tags: multiqc
+
+
+#- name: Install MultiQC from pip
```

Comment:
Remove this if we're going to fetch from Github going forth -- or are we not decided?